hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/MapTable.tpl

```diff
@@ -85,13 +85,12 @@
           </table>
         </div>
         <script type="text/javascript">
-          CRM.$(function($) {
-            {if $mappingDetailsError }
-              $('#saveDetails').show();
-            {else}
-              $('#saveDetails').hide();
-            {/if}
-          });
+          {if $mappingDetailsError }
+            cj('#saveDetails').show();
+          {else}
+            cj('#saveDetails').hide();
```

Comment:
I actually copied this from the core Contact MapTable.tpl file .... the difference between cj and $ is that there is two copies of Jquery included , one shipped with drupal and the other with civicrm so to avoid the conflict the decided to create an alias for civicrm juqery copy so $ become cj.